### PR TITLE
PAR-1034: Add Tota11y web accessibility tool to site

### PR DESCRIPTION
**Pull request by stemount**
widget enabled for devs to check accessibility as well.

### CODE REVIEW

File: tests/src/features/0-user-journey-primary-authority-new-direct-partnership.feature [hunk 1]
```diff
@@ -1,10 +1,13 @@
-@ci
 Feature: New Direct Partnership For PA To Approve
 
+    @ci @PAR1034
     Scenario: New Direct Partnership
+        Given I open the url "/user/login"
+        And I run tota11y against the current page
 
+    @ci
+    Scenario: New Direct Partnership
```
Comment (jon-edery) [inline tests/src/features/0-user-journey-primary-authority-new-direct-partnership.feature]: Not sure we should be adding tests for a dev tool to be part of the regression tests.  Can we check what the testing policy should be here.

Comment (kalpaitch) [reply tests/src/features/0-user-journey-primary-authority-new-direct-partnership.feature]: Agreed, it should be able to run as a separate set of tests.

**Comment by jon-edery** [comment]
Apart from the issue around testing looks gd 👍

File: tests/src/features/0-user-journey-primary-authority-new-direct-partnership.feature (hunk 1 above)
Comment (jaffamonkey) [reply tests/src/features/0-user-journey-primary-authority-new-direct-partnership.feature]: Will omit the step for now. Can run as separate job.

**Pull request merged**